radix-engine-tests/tests/kernel/kernel_open_substate.rs

```diff
@@ -35,21 +35,22 @@ pub fn test_open_substate_of_invisible_package_address() {
     let mut database = InMemorySubstateDatabase::standard();
     let scrypto_vm = ScryptoVm::<DefaultWasmEngine>::default();
     let native_vm = DefaultNativeVm::new();
-    let vm = Vm {
+    let vm_init = VmInit {
         scrypto_vm: &scrypto_vm,
-        native_vm: native_vm.clone(),
+        native_extension: NoExtension,
     };
-    Bootstrapper::new(NetworkDefinition::simulator(), &mut database, vm, false);
+    Bootstrapper::new(NetworkDefinition::simulator(), &mut database, vm_init, false);
 
     // Create kernel
     let mut id_allocator = IdAllocator::new(executable.intent_hash().to_hash());
-    let mut system = SystemConfig {
+    let mut system = System {
         blueprint_cache: NonIterMap::new(),
         auth_cache: NonIterMap::new(),
         schema_cache: NonIterMap::new(),
-        callback_obj: Vm {
+        callback: Vm {
```

Comment:
Rename rename `callback` to `vms`?

Reply:
The Vm object should still be opaque to the System.